srcpkgs/hplip/template

```diff
@@ -1,6 +1,7 @@
 # Template file for 'hplip'
+# Must be in sync with hplip-plugin package
```

Comment:
Drop this comment. The hplip-plugin package doesn't exist. Even if it is accepted, we shouldn't demand that people keep a non-free, restricted package in sync. If HP delays or abandons releases of the plug-in package, I'd rather people update this regularly than wait on a package fee people are likely to discover and use anyway.

If the versions really must be tied, the right thing to do is add a version-restricted dependency in the hplip-plugin package.

Reply:
I'll remove it.